dev-cpp/gtest/files/gtest-1.8.1-python3.patch

```diff
@@ -0,0 +1,122 @@
+Upstream PR: https://github.com/google/googletest/pull/1839
```

Comment:
To be honest, I'm not sure I like the idea of having more patches to support something that's not strictly necessary. Wouldn't it be cleaner to just kill python3 and wait for upstream to start supporting it properly?

Reply:
OK.